### PR TITLE
Update QQ to 8.9.6 and fix the wrong desktop profile installation.

### PR DESCRIPTION
**Pull request by hillwoodroc**
The present installation can not create the correct desktop profile, it shoud be fixed by this pull request.

### CODE REVIEW

File: src/winetricks [hunk 1]
```diff
@@ -15724,6 +15724,9 @@ load_qq()
     if w_workaround_wine_bug 37680 "Disable txplatform.exe to work around QQ can't be quit cleanly"; then
         w_override_dlls disabled txplatform.exe
     fi
+
```
Comment (austin987) [inline src/winetricks:15727]: trailing whitespace

Comment (hillwoodroc) [reply src/winetricks]: Fixed, please check.

**Comment by austin987** [comment]
Still fails travis (shellcheck).

FYI, you can run `./tests/shell-checks` locally to replicate what travis is doing.

**Comment by hillwoodroc** [comment]
> Still fails travis (shellcheck).
> 
> FYI, you can run `./tests/shell-checks` locally to replicate what travis is doing.

I don't understant the shell check. But this update works fine here.

```
hillwood@hillwood-opensuse:~/winetricks> ./tests/shell-checks 
+ '[' '!' -f Makefile ']'
++ mktemp -d
+ temp=/tmp/tmp.1ZHXfoAeuP
+ trap 'rm -fr "$temp"' EXIT
+ main
+ '[' -d .git ']'
++ git ls-files
++ xargs file
++ grep -e 'POSIX shell script'
++ cut -d : -f1
+ files_to_check='misc/bisect-wine-winetricks.sh
misc/convert_sha1_sha256.sh
misc/travis-install-bashate-deb.sh
misc/travis-install-bashate-pip.sh
src/linkcheck.sh
src/release.sh
src/uniquefiles.sh
src/winetricks
tests/self-update-test
tests/shell-checks
tests/winetricks-dotnet-grid-test
tests/winetricks-test'
+ test_linkcheck
+ grep '^ *w_download ' src/winetricks
+ grep -E 'ftp|http'
+ grep -v w_linkcheck_ignore=1
+ sed 's/^ *//'
+ tr -d '\\'
+ cut -d ' ' -f2
+ grep '\$'
+ echo 'linkcheck checks passed'
linkcheck checks passed
+ test_travis_yml
++ command -v yq
+ '[' '!' -x '' ']'
+ echo 'warning: yq unavilable, skipping .travis.yml validation'
warning: yq unavilable, skipping .travis.yml validation
+ for shellscript in $files_to_check
+ test_formatting
+ grep -n -r '[[:blank:]]$' misc/bisect-wine-winetricks.sh
++ printf '\t'
+ grep -n '	' misc/bisect-wine-winetricks.sh
+ grep -n -w -e '  do$' -e '^do$' misc/bisect-wine-winetricks.sh
+ grep -n -w -e '  then$' -e '^then$' misc/bisect-wine-winetricks.sh
+ test_bashate
++ command -v bashate
+ '[' '!' -x '' ']'
+ '[' '' = osx ']'
++ command -v bashate
+ bashate=
+ rm -fr /tmp/tmp.1ZHXfoAeuP
```

**Comment by austin987** [comment]
You may have an old version of shellcheck; you can check against the latest version at
https://www.shellcheck.net/

```
$ shellcheck myscript
 
Line 2:
echo Exec=env WINEPREFIX="$WINEPREFIX" "$WINE" \"$W_PROGRAMS_X86_WIN/Tencent/QQ/bin/QQScLauncher.exe\" >> "$HOME/.local/share/applications/wine/Programs/腾讯软件/QQ/腾讯QQ.desktop"
                                                 ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
echo Exec=env WINEPREFIX="$WINEPREFIX" "$WINE" \""$W_PROGRAMS_X86_WIN"/Tencent/QQ/bin/QQScLauncher.exe\" >> "$HOME/.local/share/applications/wine/Programs/腾讯软件/QQ/腾讯QQ.desktop"
```

**Comment by hillwoodroc** [comment]
> You may have an old version of shellcheck; you can check against the latest version at
> https://www.shellcheck.net/
> 
> ```
> $ shellcheck myscript
>  
> Line 2:
> echo Exec=env WINEPREFIX="$WINEPREFIX" "$WINE" \"$W_PROGRAMS_X86_WIN/Tencent/QQ/bin/QQScLauncher.exe\" >> "$HOME/.local/share/applications/wine/Programs/腾讯软件/QQ/腾讯QQ.desktop"
>                                                  ^-- SC2086: Double quote to prevent globbing and word splitting.
> 
> Did you mean: (apply this, apply all SC2086)
> echo Exec=env WINEPREFIX="$WINEPREFIX" "$WINE" \""$W_PROGRAMS_X86_WIN"/Tencent/QQ/bin/QQScLauncher.exe\" >> "$HOME/.local/share/applications/wine/Programs/腾讯软件/QQ/腾讯QQ.desktop"
> ```

Fixed. It's ok now.

**Pull request merged**